client/src/services/Api.js

```diff
@@ -5,7 +5,8 @@ export default () => {
   return axios.create({
     baseURL: `http://localhost:3000/`,
```

Comment:
Although not touched in this branch, I think this `3000` should be replaced with the environment variable.